LIFECYCLE.md

```diff
@@ -0,0 +1,59 @@
+# Lifecycle Maintenance of Devfile Registry Stacks
+
+The purpose of this document is to clarify the roles and responsibilities for maintaining the devfile stacks and samples present in the [devfile Community registry](https://github.com/devfile/registry).
+Devfiles are intended to be used by developers to build their applications with tooling clients that support the devfile spec. As a result, it’s important that stack providers avoid introducing breaking changes or allowing security vulnerabilities to go unremediated which would result in a degradation of our end users applications.  In short, we need to ensure our devfiles are trusted enough to be used. In order to achieve this, there is a level of shared responsibility in all of the roles defined below.
+
+
+| Role                    | Description                                                                                                                             |
+|:------------------------|:----------------------------------------------------------------------------------------------------------------------------------------|
+| `Devfile Team`          | The team that is responsible for onboarding vendor devfiles for the community registry.                              |
+| `Stack/Sample Provider` | The person or team that is responsible for developing the devfile stack or sample that is available for sharing on our devfile registry. |
+| `Tooling Clients`       | These are the tools that support building devfile-based applications                                                                    |
```

Comment:
If Eclipse Che and odo are among such clients, then the description is not correct. You don't build devfile based application with Che or odo.
```suggestion
| `Tooling Clients`       | These are the tools that use devfiles                                                                    |
```